### PR TITLE
Adding HW8

### PR DESCRIPTION
**Pull request by olark1d**
Added HomeWork 8, waiting for feedback:)

### CODE REVIEW

File: HOMEWORK_8/EX5/RecursiveFunctions.cpp [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+int number = 1;
+void printFromSmallerToBiggerByRecursion(int count)
+{
+    if(number > count)
+        return;
+
+    std::cout<<number<<std::endl;
+    number++;
+    printFromSmallerToBiggerByRecursion(count);
+}
```
Comment (bohdanpc) [inline HOMEWORK_8/EX5/RecursiveFunctions.cpp]: can be implemented without additional `number` variable

File: HOMEWORK_8/EX4.CPP [hunk 2]
```diff
@@ -0,0 +1,33 @@
+#include <iostream>
+
+const float PI = 3.14;
```
Comment (bohdanpc) [inline HOMEWORK_8/EX4.CPP]: prefer using predefined pi value, for example M_PI from cmath

File: HOMEWORK_8/EX5/RecursiveFunctions.cpp (hunk 1 above)
Comment (olark1d) [reply HOMEWORK_8/EX5/RecursiveFunctions.cpp]: I hope I understood correctly

File: HOMEWORK_8/EX4.CPP (hunk 2 above)
Comment (olark1d) [reply HOMEWORK_8/EX4.CPP]: done

File: HOMEWORK_8/EX5/RecursiveFunctions.cpp [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+void printFromSmallerToBiggerByRecursion(int count, int number)
+{
+    if(number > count)
+    return;
+
+    std::cout<<number<<std::endl;
+    printFromSmallerToBiggerByRecursion(count, number + 1);
+}
```
Comment (bohdanpc) [inline HOMEWORK_8/EX5/RecursiveFunctions.cpp]: You don't need additional variables at all in any form. Neither inside of the function nor as a parameter.

Comment (olark1d) [reply HOMEWORK_8/EX5/RecursiveFunctions.cpp]: done x2

**Pull request merged**